lib/rubocop/cop/rspec/variable_definition.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that variables are defined using symbols or strings.
```

Comment:
Reading this description in our documentation, I would think “do they mean local variables or instance variables?”. It’s not immediately obvious that by “variables” we mean the ones you define by calling `let`, `subject`, etc.

Reply:
I agree that we need better wording. Strictly speaking, it's a memorized helper method and not a variable